Tests/LibWeb/Text/expected/HTML/image-bitmap-from-invalid-types-no-crash.txt

```diff
@@ -1,7 +1,7 @@
 Blob              [Success]: [object ImageBitmap]
```

Comment:
Please update the commit message w/ regard to HTMLOrSVGElement.idl

Reply:
done